### ISSUE TITLE
We can add the current local time too of the selected location.

### ISSUE THREAD

**Issue by Unnati2603** [Ahsankhalid618/react-weather-app#40]
The 1st block can show local time along with other details. Please Assign me this if you are interested in this feature 😃 @Ahsankhalid618

**Comment by Ahsankhalid618** [comment]
@Unnati2603 Good idea, but just so you know, the Ist block will have a graph of 24-hour weather details (Image attached below). and still, if you want to add the local time then make sure the UI should be appealing. Let me know if you are up for this.

**Comment by Unnati2603** [comment]
I have a few ideas @Ahsankhalid618 
1.  ![image](https://github.com/user-attachments/assets/6ecb7e8d-880b-4793-8084-2dd1619d6e53)
We can try this? How does this look?

2. or we can add it below the date

3. next to name of place, in a smaller font

4. at the right most corner like this:
![image](https://github.com/user-attachments/assets/1805e87b-1257-40d3-8af6-ad69e76cdba5)

**Comment by Ahsankhalid618** [comment]
@Unnati2603 I think, it will look good on the right corner.

**Comment by Unnati2603** [comment]
Ok, can you assign me?
and please add the hacktoberfest tag. I will try to complete it as soon as possible
